lib/rails_cookie_overflow/railtie.rb

```diff
@@ -4,7 +4,7 @@ module RailsCookieOverflow
   # Railtie to set the middleware during boot
   class Railtie < ::Rails::Railtie
     initializer "rails_cookie_overflow.insert_middleware" do |app|
-      app.config.middleware.insert_before "ActionDispatch::Cookies", "RailsCookieOverflow::Middleware"
+      app.config.middleware.insert_before ActionDispatch::Cookies, RailsCookieOverflow::Middleware
```

Comment:
can we also change the gemspec to support only rails >= 5/5.2 (whichever version changed this)

Reply:
It is supported in 4.2 also, so no changes needed in gemspec